ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/DC1-BL1A.cfg

```diff
@@ -249,6 +255,9 @@ router bgp 65104
    neighbor 192.168.255.4 peer group EVPN-OVERLAY-PEERS
    neighbor 192.168.255.4 remote-as 65001
    neighbor 192.168.255.4 description DC1-SPINE4
+   neighbor 192.168.255.16 peer group EVPN-OVERLAY-CORE
+   neighbor 192.168.255.16 remote-as 65106
+   neighbor 192.168.255.16 description DC1-BL2A
    redistribute connected route-map RM-CONN-2-BGP
    !
    vlan-aware-bundle Tenant_A_WAN_Zone
```

Comment:
We are missing the following for some reason, looks like it's not being generated

          - domain: "remote"
            rd: "{{ network_services_data.rd_admin_subfield }}:{{ vlan_rt }}"
            route_target: "{{ network_services_data.rt_admin_subfield or vlan_rt }}:{{ vlan_rt }}"

Reply:
Yaay!